README.md

```diff
@@ -37,4 +37,27 @@ helm install aws-driver aws-driver-0.0.1.tgz --set docker.image=[registry]/aws-d
 The AWS driver expects an `AWS` deployment location with the following properties:
 
 * AWS_ACCESS_KEY_ID: the AWS Access Key Id of your AWS accouunt
-* AWS_SECRET_ACCESS_KEY: the AWS Secret Access Key of your AWS accouunt
\ No newline at end of file
+* AWS_SECRET_ACCESS_KEY: the AWS Secret Access Key of your AWS accouunt
+
+## Scalability
+
+   With Gunicorn support, aws-driver can handle huge traffic by configuring no of processes and threads.
```

Comment:
Is this repetation of the below line 50?

Reply:
@rajeevbiswal  forgot to delete that.. Just pushed the change